src/libs/dataUseTranslation.js

```diff
@@ -278,19 +283,18 @@ export const DataUseTranslation = {
      *
      * Tracing this through to Ontology, both refer to http://purl.obolibrary.org/obo/DUO_0000011 which is POA
      */
-    if (darInfo.poa || darInfo.population || darInfo.populationMigration) {
+    if (darInfo.poa) {
```

Comment:
Similar to the above comment, we still need to check for `darInfo.population` here.